src/test/java/pages/LoginPage.java

```diff
@@ -33,4 +34,30 @@ public LoginPage loginWith(String email, String password){
         return this;
     }
 
+    public LoginPage verifyLoginEmail(String text){
+        assertEquals(errorEmailLabel, text);
```

Comment:
Do not hide asserts into pages! it will difficult to understand another people what your test checks